### DISCUSSION TITLE
How to run Cfast in verbose mode to print on screen steps of the calculation

### DISCUSSION THREAD

**Discussion by fconsignygit** [firemodels/cfast#2143] [Q&A]
When you run Cfast from a command prompt (windows version), how can you display information on the console (of the type written to the output file for each time step) about the calculation as it is executed (and not just the total execution time at the end)?

**Comment by preneke** [comment]
Sorry for the delay. I am sorry but we don't do that anymore. You are the first person I know of who has asked. I don't know for sure off the top of my head but I don't expect it would be an easy fix. Why are you interested? Maybe there is another fix.

**Reply by fconsignygit** [reply]
Thanks for the reply, I'm interested because as part of a stochastic study I would need to run a large number of cfast runs (several days of calculations) and I wanted to make sure (from time to time) that the runs were running well and converging normally.

**Comment by gforney** [comment]
in the current cfast, pressing the F5 key will output while the simulation is running the current time and timestep the solver is using

I added a -L option to cfast (located at URL below) that outputs interim times, fire hrr's and upper layer temperatures, similar data that goes the the .out file .  note, the option needs to go after the input file as in

cfast casename.in -L

https://github.com/firemodels/test_bundles/releases/tag/CFAST_TEST

**Reply by fconsignygit** [reply]
Thanks a lot, I tested this version with the -L option and it works :-)

**Reply by fconsignygit** [reply]
I have a subsidiary question, this version of cfast (CFAST_TEST in the URL sent in your repy) seems to run much faster than the initial version i add (v7.7.4.0), is this normal?

**Comment by gforney** [comment]
Not sure. We're building with different versions of the compiler now. Can
you upload the case you are getting faster times for.  Ill build cfast  774
with current compilee to see if that is the cause

**Reply by fconsignygit** [reply]
Sorry for the late reply.
[N&N1_Test - in.txt](https://github.com/user-attachments/files/18684161/N.N1_Test.-.in.txt)
It runs in 3.09s with Cfast_test (23655 Time stes) and 36.1s (166717 time steps) with previous installed cfast...
